test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/ExpressionHelperTest.cs

```diff
@@ -12,6 +12,79 @@ public class ExpressionHelperTest
     {
         private readonly ExpressionTextCache _expressionTextCache = new ExpressionTextCache();
 
+        public static IEnumerable<object[]> ExpressionAndTexts
+        {
+            get
+            {
+                var i = 3;
+                var value = "Test";
+                var Model = new TestModel();
+                var key = "TestModel";
+                var myModels = new List<TestModel>();
+                var __DisplayClass = new List<TestModel>();
```

Comment:
Avoid writing code that violates the c# rules. Expressions such as `model => myModels[i].SelectedCategory` should be complex enough to hit compiler-generated field names. Set a breakpoint on relevant line and confirm instead of including this test.